src/components/Items/Items.js

```diff
@@ -2,13 +2,17 @@ import React, { Component, Fragment } from 'react';
 import { connect } from 'react-redux';
 import { Redirect } from 'react-router-dom';
 import AddItem from '../AddItem';
+import { Form, Text } from 'react-form';
```

Comment:
This import causes React to fail. You don't appear to be using it, and haven't added it to `package.json`.